service/src/main/java/uk/gov/hmcts/reform/fpl/enums/ReturnedApplicationReasons.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+public enum ReturnedApplicationReasons {
+    INCORRECT("Application Incorrect"),
+    INCOMPLETE("Application Incomplete"),
+    CLARIFICATION_NEEDED("Clarification Needed");
+
+    private final String label;
+
+    ReturnedApplicationReasons(String label) {
+        this.label = label;
+    }
```

Comment:
think you can remove this constructor if you do 

@Getter
@RequiredArgsConstructor